package.json

```diff
@@ -20,6 +20,8 @@
   "devDependencies": {
     "@types/mocha": "^5.2.5",
     "@types/node": "^10.5.7",
+    "@types/debug": "^4.1.3",
+    "debug": "^4.1.1",
```

Comment:
I believe debug is needed at runtime when published, right? This should go in dependencies?